public/Git_Card_UI/index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
```

Comment:
This has to be a React Component (which means HTML within a JavaScript Component, JSX), like all the other components as well.